demo/src/components/demos/geojson-demo.js

```diff
@@ -32,28 +53,29 @@ export default class GeoJsonDemo extends Component {
   static renderInfo(meta) {
 
     const legendCount = 5;
-    const legendBlocks = (new Array(legendCount)).fill(0).map((d, i) => {
-      const color = colorScale(i / (legendCount - 1) * MAX_AGE).map(Math.round);
-      return `rgb(${color.join(',')})`;
+    const legends = new Array(legendCount).fill(0).map((d, i) => {
```

Comment:
Total nit: I think this type of expression can be written without creating a temporary array.
```
const legends = new Array(legendCount).fill(0).forEach((d, i, array) => {
  array[i] = i / ...;
});
```
For such a short array, staying with map and using the "body-less" form could be nicer.
```
const legends = new Array(legendCount).fill(0).map((d, i) => i / (legendCount - 1) * 2 - 0.5);
```